registry.yaml

```diff
@@ -4,6 +4,15 @@
 # should build pages for, and indicates metadata such as tags, creation date and
 # authors for each page.
 
+- title: GPTs for Business 
+  path: articles/GPTs_for_Business.md
+  date: 2023-12-10
+  authors:
+    - sabrina-de
+  tags:
+    - ChatGPT
+    - Actions
+    - GPTs
```

Comment:
We don't have any of these tags yet afaik – maybe set it as just "GPTs" for now while we add it